docs/templates/plugin.rst.j2

```diff
@@ -123,19 +123,24 @@ Parameters
                 </td>
                 {# default / choices #}
                 <td>
+		    {# Trun string like "yeS", "No" or "TruE", "fAlse" to boolean #}
+                    {% if value.default is string %}
+                        {% if value.default.lower() == 'yes' %}
```

Comment:
I bet you could use ` in ('yes', 'true', '1'...)`

Reply:
Ok, I will switch to in() keyword.